src/libutil/serialise.hh

```diff
@@ -153,12 +153,13 @@ struct FdSource : BufferedSource
 {
     int fd;
     size_t read = 0;
+    std::string endOfFileError{"unexpected END-of-file"};
```

Comment:
nit (capital "END" seems weird?)

```suggestion
    std::string endOfFileError{"unexpected end-of-file"};
```

Reply:
Oops, that was a debug thingy. Thanks for catching that.